### PR TITLE
Added Dutch

### PR DESCRIPTION
**Pull request by Haroenv**
Validation.php isn't fully translated yet though.

### CODE REVIEW

File: frontend/app/lang/nl/languages.php [hunk 1]
```diff
@@ -0,0 +1,73 @@
+<?php
```
Comment (joshlemer) [inline frontend/app/lang/nl/languages.php]: @Haroenv did you forget to translate this file? :)

File: frontend/app/lang/nl/validation.php [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+	"array"                => "De :attribute mag enkel een array bevatten. ",
+	"before"               => "De :attribute moet voor :date zijn.",
+	"between"              => array(
+		"numeric" => "De :attribute moet tussen :min en :max zijn.",
+		"file"    => "De :attribute moet tussen :min en :max kilobytes zijn.",
+		"string"  => "De :attribute moet tussen :min en :max karakters zijn.",
+		"array"   => "De :attribute moet tussen :min en :max dingen hebben.", /*Not the best translation*/
+	),
+	"boolean"              => "De :attribute moet \"true\" of \"false\" zijn.",
+	"confirmed"            => "De :attribute va",
```
Comment (joshlemer) [inline frontend/app/lang/nl/validation.php]: @Haroenv Also many of these

**Comment by Haroenv** [comment]
Oh, I also had to translate the languages? I can do that ASAP, and I didn't have time to finish `validation.php` yet. I can do that tomorrow.

**Comment by Haroenv** [comment]
@joshlemer those files are also translated now.

**Comment by joshlemer** [comment]
:+1:

**Comment by Haroenv** [comment]
You're welcome!

**Pull request merged**